apps/etterlatte-saksbehandling-ui/client/src/components/behandling/revurderingsoversikt/sluttbehandlingUtland/SluttbehandlingUtland.tsx

```diff
@@ -29,7 +34,27 @@ export default function SluttbehandlingUtland({ sakId, revurderingId }: { sakId:
     })
   }, [])
 
+  const validerSkjema = () => {
+    const feilkoder: Set<string> = new Set([])
+    if (landMedDokumenter.find((landmedDokument) => landmedDokument.landIsoKode === undefined)) {
+      feilkoder.add('MANGLER_LANDKODE')
+    }
+    if (landMedDokumenter.find((landMedDokument) => landMedDokument.dokumenter.length === 0)) {
+      feilkoder.add('LAND_MANGLER_DOKUMENTER')
+    } else {
+      landMedDokumenter.forEach((landMedDokument) => {
+        if (landMedDokument.dokumenter.find((e) => e.dokumenttype === undefined)) {
+          feilkoder.add('MANGLER_DOKUMENTTYPE')
+        }
+        if (landMedDokument.dokumenter.find((e) => e.dato === undefined)) {
+          feilkoder.add('MANGLER_DATO')
+        }
+      })
+    }
+    setFeilkoder(feilkoder)
+  }
```

Comment:
`=== undefined` kan byttes ut med `!` (foran). Da sjekker du etter undefined, null, tom string, false, 0, m.m.

Reply:
Ja sant det, siden det dekker flere caser er det kanskje bedre? 
Setter alt auto til `undefined` for de feltene men.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2485/commits/2aca7abc08ca638a010106de4e49d02c74262e39